lib/build/bundle.js

```diff
@@ -127,6 +127,39 @@ exports.Bundle = class {
         let currentFile = files[i];
         let sourceMap = buildOptions.sourcemaps ? currentFile.sourceMap : undefined;
 
+        function fileIsDependency(file) {
+          return file
+            && file.path
+            && file.path.indexOf("node_modules") >= 0;
+        }
+
+        function acquireSourceMapForDependency(file) {
+          if (!file || !file.path) {
+            return;
+          }
+
+          let parsedPath = path.parse(file.path);
+
+          let converter = Convert.fromMapFileSource(
```

Comment:
`fromMapFileSource` fails with some files, because the bundle has appended a `define` statement to the end of the file.